common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -640,14 +701,21 @@ def get_problem_html(self, encapsulate=True):
             check_button = False
             check_button_checking = False
 
-        problem_is_timed = self.minutes_allowed > 0
+        # For timed problems
+        now = datetime.datetime.now(UTC())
+        problem_has_finished = False
+        seconds_left = -1
+        end_time_to_display = now + datetime.timedelta(minutes=self.minutes_allowed)
+
+
+        if self.is_timed_problem() and self.time_started:
+                end_time_to_display = self.time_started + \
+                    datetime.timedelta(minutes=self.minutes_allowed)
+                problem_has_finished = end_time_to_display >= now
+                time_left = end_time_to_display - now
+                seconds_left = (time_left).total_seconds()
```

Comment:
fix formatting of this block:  too much indentation, unnecessary line continuation